openbas-front/src/components/common/queryable/filter/FilterUtils.tsx

```diff
@@ -103,6 +99,10 @@ export const availableOperators = (propertySchema: PropertySchemaDTO) => {
   if (propertySchema.schema_property_has_dynamic_value && !propertySchema.schema_property_type_array) {
     return ['contains', 'not_contains', 'empty', 'not_empty'];
   }
+  // Array
+  if (propertySchema.schema_property_values || propertySchema.schema_property_type_array) {
+    return ['contains', 'not_contains', 'empty', 'not_empty'];
+  }
```

Comment:
Why this changes ?
Now, we can't filter with empty and is empty for several property
![image](https://github.com/user-attachments/assets/93279b3e-61fe-4fc8-a9ee-d948bbcc8dd9)

Additionally, this breaks compatibility with existing filters in local storage that used other operators.
This is a big functional change.

Reply:
I changed the order of the conditions because this one is less strict than the others: In this cse we have: 
 // Array
  if (propertySchema.schema_property_type_array **|| propertySchema.schema_property_values**) {
    return ['contains', 'not_contains', 'empty', 'not_empty'];
  }
  // Enum & not array
  **if (propertySchema.schema_property_values** && !propertySchema.schema_property_type_array) {
    return ['eq'];
  } .. For the empty case in an enumerated type, I’d prefer to add this option directly within the enumeration.